sdk/lib/crt/string/wcs.c

```diff
@@ -257,7 +257,8 @@ INT CDECL wctomb( char *dst, wchar_t ch )
 /*********************************************************************
  * wcsrtombs_l (INTERNAL)
  */
-static size_t CDECL wcsrtombs_l(char *mbstr, const wchar_t **wcstr, size_t count, _locale_t locale)
+static size_t CDECL wcsrtombs_l(char *mbstr, const wchar_t **wcstr,
+            size_t count, _locale_t locale)
 {
     MSVCRT_pthreadlocinfo locinfo;
     size_t tmp = 0;
```

Comment:
It is interesting to note that in this function Wine does no check for wcstr being NULL.